.github/workflows/main.yml

```diff
@@ -1,77 +0,0 @@
-name: Test, build, and deploy
```

Comment:
I dont think we can remove this quite yet, there is a lot more in here

Reply:
We are not going to merge , before everything is done - like staging deployments.